File: Service/ClientCare/RMA.Service.ClientCare.Contracts/Entities/Policy/MoveSchemeCase.cs
using System.Collections.Generic;

namespace RMA.Service.ClientCare.Contracts.Entities.Policy
{
    public class MoveSchemeCase
    {
        public string Code { get; set; }
        public int SourcePolicyId { get; set; }
        public string SourcePolicyNumber { get; set; }
        public int DestinationPolicyId { get; set; }
        public string DestinationPolicyNumber { get; set; }
        public List<int> PolicyIds { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/clientcare/src/app/policy-manager/shared/entities/move-scheme-case.ts
import { BaseClass } from 'projects/shared-models-lib/src/lib/common/base-class';

export class MoveSchemeCase extends BaseClass {
    code: string;
    sourcePolicyId: number;
    sourcePolicyNumber: string;
    destinationPolicyId: number;
    destinationPolicyNumber: string;
    policyIds: number[];
}
